src/authentication/raw-deployment-authorization-spec.ts

```diff
@@ -89,4 +89,17 @@ describe('authorization for raw deployments', () => {
     expect(handlerStub).to.not.have.been.called;
 
   });
+  it('should redirect to login screen for unauthenticated deployments', async () => {
+    // Arrange
+    const { server, authentication, handlerStub } = await arrange('userHasAccessToProject', true);
+    // Act
+    const response = await server.inject({
+      method: 'GET',
+      url: `http://deployment-abcdef-1-1.foo.com`,
+    });
+    // Assert
+    expect(response.statusCode).to.eq(302);
+    expect(authentication.userHasAccessToProject).to.not.have.been.called;
+    expect(handlerStub).to.not.have.been.called;
```

Comment:
Ok so this was actually a painful point. One of the reasons for going with the separate subdomain/vhost strategy in #174 was the way the raw deployments are served here. Since the request is internally routed to a new path, the original requested path was not trivial to uncover. TLDR; the redirect location is not what it should be, so better not not to test it. Fixed in #174.